packages/ws-server/src/webSocketServer.ts

```diff
@@ -169,6 +174,20 @@ app.ws.use(async (ctx) => {
             connectionIdPrefix,
           );
           break;
+
+        case WS_CONSTANTS.METHODS.ETH_GET_TRANSACTION_BY_HASH:
+          response = await handleEthGetTransactionByHash(
+            ctx,
+            params,
+            logger,
+            relay,
+            request,
+            method,
+            socketIdPrefix,
+            requestIdPrefix,
+            connectionIdPrefix,
```

Comment:
question, what is the difference between socketId and connectionId.

Reply:
This is actually a good question they are actually both the same. I have made an update to remove the socketId in the next PR